app/views/sidebar/_guidelines.html.haml

```diff
@@ -39,36 +36,24 @@
           To protect yourself and others:
         %ul
           %li
-            DO NOT stand in the street while cleaning. Clean the storm drain from the sidewalk.
+            DO NOT stand in the street while cleaning.
           %li
-            DO NOT stand in the street while cleaning. Clean the storm drain from the sidewalk.
+            DO NOT pick up trash that is inside of the storm drain or too close to the roadway.
           %li
-            DO NOT clean the drain during dark hours or inclement weather.
+            DO NOT clean the drain during dark hours or inclementweather.
           %li
-            DO NOT volunteer or perform any of these activities under the influence of alcohol and drugs.
+            DO NOT volunteer or perform any of these activities under the influence of alcohol and/ordrugs.
           %li
-            DO NOT pick up weapons or bullet casings. If found, call the non-emergency number at 441-5610.
+            DO NOTpick up weapons, bullets, or ammunition. If found, call the Non-Emergency Number at 441-5610to report its location
           %li
-            DO NOT pick up syringes or hypodermic needles with bare hands. Wearing gloves, an adult should pick up a needle with a pair of pliers or similar device; drop the needle in a hard-wall container such as a metal can or plastic bottle. Seal the container and place in the trash. (As instructed by the Norfolk Health Department)
-          %li
-            DO NOT pick up materials that appear to be hazardous or dangerous. Report graffiti, dumping, or spills to the Norfolk Cares Line at 664-6510.
-          %li
-            Although rare, “bottle bombs” made of plastic bottles do exist. DO NOT pick up any partially filled, closed bottles that appear to have metal or hazardous liquids inside. If a suspicious bottle is found, back away from it at least 100 feet, prevent any other individuals from entering the area, and finish your clean up. Be sure to call 911 to inform the police of what you’ve found and where you found it.
+            DO NOTpick up materials that appear to be hazardous or dangerousincluding, but not limited to, needles, syringes, and bottle bombs. Although rare, “bottle bombs” made of plastic bottles do exist.Do notpick up any partially filled, closed bottles that appear to have metal or hazardous liquids inside. If a suspicious bottle is found, back away from it at least 100 feet, prevent any other individuals from enteringthe area, and call 911to inform the police of what you’ve found and where you found it. Report graffiti, dumping, or spills to the Norfolk Cares Line at 664-6510.
```

Comment:
missing spaces at various sections in the sentence 

DO NOTpick 
and exist.Do 
911 to
enteringthe 
etc